Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -458,37 +458,34 @@ try {
                 # Update the files
                 # Calculate the release notes, while updating
                 $releaseNotes = ""
-                try {
```

Comment:
Because I couldn't find a good reason for swallowing errors here.
Basically the try/catch would mean that you could end up having 50% of your files updated and the rest not.